src/extensions/default/CodeFolding/main.js

```diff
@@ -284,10 +268,118 @@ define(function (require, exports, module) {
     }
 
     /**
+      * Utility function to check if the code folding menu options exist in the menu
+      */
+    function menuExists(id) {
+        var viewMenu = Menus.getMenu(Menus.AppMenuBar.VIEW_MENU);
+        return Menus.getMenuItem(viewMenu._getMenuItemId(id));
+    }
+
+    /**
+      * Create the codefolding menu items and register key bindings if they dont already exist
+      */
+    function createMenuItems() {
+        //register commands
+        if (!CommandManager.get(COLLAPSE_ALL)) {
+            CommandManager.register(Strings.COLLAPSE_ALL, COLLAPSE_ALL, collapseAll);
+        }
+
+        if (!CommandManager.get(EXPAND_ALL)) {
+            CommandManager.register(Strings.EXPAND_ALL, EXPAND_ALL, expandAll);
+        }
+
+        if (!CommandManager.get(COLLAPSE)) {
+            CommandManager.register(Strings.COLLAPSE_CURRENT, COLLAPSE, collapseCurrent);
+        }
+
+        if (!CommandManager.get(EXPAND)) {
+            CommandManager.register(Strings.EXPAND_CURRENT, EXPAND, expandCurrent);
+        }
+
+        //create menus
+        if (!menuExists(codeFoldingMenuDivider)) {
```

Comment:
Looks like the divider gets recreated every time you set the `"enabled"` pref to true at runtime.  That's kind of an edge case -- I'll see if I can address it in my PR instead of worrying about it here.